mod.ts

```diff
@@ -85,6 +85,10 @@ export interface BuildOptions {
   redirects?: Redirects;
   /** Package.json output. You may override dependencies and dev dependencies in here. */
   package: PackageJsonObject;
+  /** Package manager used to install dependencies and run npm scripts.
```

Comment:
> there may be some environments that npm isn't available in $PATH

This point is compelling.

Perhaps let's change this to something like `npmCliBinPath`? Then someone could specify an npm-like CLI like `pnpm` or `yarn` if they really wanted to, but it wouldn't be explicitly supported for now.

Reply:
`npmCliBinPath` doesn't match intention of this PR. When someones reading `npmCliBinPath`, they may just think "specifying a different path to `npm`", rather than "using another package manager".

Reply:
Alright, discussed this with others and let's leave it as "packageManager".